app/services/teacher_vacancy_authorisation.rb

```diff
@@ -52,4 +52,4 @@ def user_permissions_for_school
       user_permissions.select { |permission| permission['school_urn'] == @school_urn }
     end
   end
-end
+end
```

Comment:
I think this is showing because a previous commit on this branch removed this file and a later commit added it back to keep the old behaviour with feature flags. We can add it but it's unlikely we'd run in to the above too often